RELEASE/scripts/autoscend/iotms/mr2016.ash

```diff
@@ -1257,19 +1285,15 @@ boolean timeSpinnerAdventure()
 
 boolean timeSpinnerAdventure(string option)
 {
-	if(is_unrestricted($item[Time-Spinner]) && (item_amount($item[Time-Spinner]) > 0) && (get_property("_timeSpinnerMinutesUsed").to_int() <= 9))
+	//spend 1 minutes to Adventure Way Back in Time
+	if(timeSpinnerRemaining(true) < 1)
 	{
-		string temp = visit_url("inv_use.php?pwd=&which=3&whichitem=9104");
-		if(get_property("_timeSpinnerMinutesUsed").to_int() <= 9)
-		{
-			string[int] pages;
-			pages[0] = "inv_use.php?pwd=&which=3&whichitem=9104";
-			pages[1] = "choice.php?pwd=&whichchoice=1195&option=3";
-			if(autoAdvBypass(0, pages, $location[Noob Cave], option)) {}
-			return true;
-		}
+		return false;
 	}
-	return false;
+	string[int] pages;
+	pages[0] = "inv_use.php?pwd=&which=3&whichitem=9104";
+	pages[1] = "choice.php?pwd=&whichchoice=1195&option=3";
+	return autoAdvBypass(0, pages, $location[Noob Cave], option);
```

Comment:
could this be replaced with `run_choice(3);`

Reply:
I think the bypass was to first run the choice selection, which will redirect into a combat which then needs to be processed. Probably need to verify this change works.

Reply:
I changed
```
if(autoAdvBypass(stuff)) {}
return true;
```
into `return autoAdvBypass(stuff);`
If bypass was working before it would also work now.
The previous function was definitely structured incorrectly.

I was also suggesting that maybe we could have done away with the bypass. But I did not actually make such a change, merely discussed it.

Incidentally the entire function `timeSpinnerAdventure` is never used anywhere in autoscend. It was written but never used. So I think that is why we did not catch that the original function was broken in that it always returns true, even if it fails.

Reply:
I meant changing it to run_choice(3) probably won't work and would need to be verified, autoAdvBypass is what will call the correct combat scripts when a fight happens here.

Reply:
But if it's not used you can probably just remove it, I doubt there's anything ascension relevant there.

Reply:
> I meant changing it to run_choice(3) probably won't work and would need to be verified, autoAdvBypass is what will call the correct combat scripts when a fight happens here.

Oh I see now. Yes I totally agree. That definitely would have to be tested.